functional-tests/nightwatch.json

```diff
@@ -4,11 +4,11 @@
 		"default": {
 			"launch_url": "http://bagcupid",
 			"selenium_host": "selenium-hub",
-			"screenshots": {
-				"enabled": true,
-				"on_failure": true,
-				"path": "tests_output"
-			}
+      "screenshots": {
+        "enabled": true,
+        "on_failure": true,
+        "path": "C:/FL/MicahZoltu/HandbagsProject/MicahRepo/bags-client/functional-tests/reports"
```

Comment:
This should be a relative path.  I don't, for example, have `C:/FL/MicahZoltu` on my computer.  Also, in linux (where this is run) that isn't a valid path.